src/util/BaseLoginRouter.js

```diff
@@ -239,8 +238,10 @@ function (Okta, Backbone, xdomain, RefreshAuthStateController, Settings, Header,
         // if we get some other type of error which doesn't force a redirect,
         // we will probably be left in a bad state. I.e. old controller is
         // dropped and new controller is not rendered.
-        oldController.remove();
-        oldController.$el.remove();
+        if (oldController) {
```

Comment:
If the first controller we load errors out on fetchInitialData, we'll try to remove a controller that doesn't exist.